### DISCUSSION TITLE
搞错了

### DISCUSSION THREAD

**Discussion by Lrrrrrrt** [mikumifa/biliTickerBuy#116] [Ideas]
现在是不需要生成配置信息了吗，现在点生成配置信息就一直提示我‘当前票信息已经更改
![屏幕截图 2024-06-14 ](https://github.com/mikumifa/biliTickerBuy/assets/108092864/1a571390-d918-4e8e-8ee2-b441273bf79e)
![屏幕截图 2024-06-14 165655](https://github.com/mikumifa/biliTickerBuy/assets/108092864/b3f3b797-4236-4405-9bd9-777b13b3f1b1)
’

**Comment by mikumifa** [comment]
最新版是网址了，粘贴方便了
